apis/core/lib/domain/job/create.ts

```diff
@@ -76,12 +76,14 @@ export async function createPublishJob({
 
   const metadata = await store.getResource(project.dataset)
 
+  const targetGraph = metadata.isHiddenCube ? $rdf.namedNode(organization.publishGraph.value + '/hidden') : organization.publishGraph
```

Comment:
Just thinking out loud, whether we want this implicit or maybe add to the organisation profile. Final effect may be the same but maybe it's going to be a little more visible for VSHN et.al. which graphs are intended to be hidden.

(I realised while typing that the current ACL idea does not really need that information but who knows...)

Reply:
I moved the flag from Cube Metadata to Project.

Now I have the issue, that it fails on `<Create Project>`. The validation reports seems to be empty - what does that mean?

![image](https://github.com/zazuko/cube-creator/assets/877232/c4860ce2-ad3e-4a60-b3b1-679dc14fdaa8)